src/dal/views.py

```diff
@@ -55,6 +55,7 @@ def has_more(self, context):
 
     def get_result_value(self, result):
         """Return the value of a result."""
+        # return six.text_type(result.pk)
```

Comment:
Right, don't worry about it, try to add your own regression test for this use case ;)